README.md

```diff
@@ -46,8 +46,12 @@ Verifiable Presentations (VP) are used to present claims along with cryptographi
 
 There are two credential formats to VCs and VPs: JSON or JSON-LD. There are also two proof formats to VCs and VPs: JWT and Linked Data Proofs. Each of those formats has different properties and capabilites and each of them comes with different proof types. Proof formats are agnostic to the credential format chosen. However, the JSON credential format is commonly used with JSON Web Signatures (https://www.w3.org/TR/vc-data-model/#json-web-token). JSON-LD is commonly used with different kinds of Linked Data Proofs and JSON Web Signatures (https://www.w3.org/TR/vc-data-model/#json-ld). 
 
-This specification defines standard claims and a separate artifact to allow implementations to support all beforementioned assertion and proof formats. These claims can be used with any OpenID Connect Flows: as JWTs such as ID tokens, or as sets of JSON claims such as UserInfo Endpoint responses. 
+This specification introduces the following mechanisms to exchange verifiable credentials objectes between OpenID OPs and RPs.
 
+* JWT claims (`verifiable_presentations` and `verifiable_credentials`) used as genetic container objects to embed verifieble credential objects into ID tokens or userinfo responses.
+* New tokens types VP Token and VC Token contain verifiable credentials objectes
```

Comment:
I would propose leave out VC token and anything related to VC out of the text as suggested during the last SIOP special call. Putting VC directly into the ID Token might lead to ID Token being treated as a new wrapper for VC competing with VP, and might allow OP to return a VC without a proof that it is a subject to whom VC was issued actually presenting it.

But it may also be a good idea to keep this option and take it out if it will not be used.

Reply:
will remove vc stuff